docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -226,7 +226,7 @@ To start a Keycloak server, use the following Docker command:
 docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8543:8443 -v "$(pwd)"/config/keycloak-keystore.jks:/etc/keycloak-keystore.jks quay.io/keycloak/keycloak:{keycloak.version} start  --hostname-strict=false --https-key-store-file=/etc/keycloak-keystore.jks
 ----
 
-where `keycloak.version` must be `23.0.0` or later and the `keycloak-keystore.jks` can be found in https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/keycloak-keystore.jks[quarkus-quickstarts/security-keycloak-authorization-quickstart/config].
+where `keycloak.version` must be `24.0.0` or later and the `keycloak-keystore.jks` can be found in https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/keycloak-keystore.jks[quarkus-quickstarts/security-keycloak-authorization-quickstart/config].
```

Comment:
we are on 25 now

Reply:
I wonder if Keycloak authorization API is backwards compatible, because if yes, it's alright.

Reply:
https://github.com/quarkusio/quarkus/pull/41791

Reply:
@michalvavrik  I'm handling it separately in PR https://github.com/quarkusio/quarkus/pull/41791.  Please let me know which version is valid for 3.8. Thank you.

Reply:
@rolfedh 24.0.0 or later

Reply:
Handled by:

- https://github.com/quarkusio/quarkus/pull/41791
- https://github.com/quarkusio/quarkus/pull/41792